programs/clearing_house/src/controller/lp.rs

```diff
@@ -25,13 +25,16 @@ pub fn settle_lp_position(
     let upnl = update_lp_position(position, &metrics)?;
     update_unsettled_pnl(position, market, upnl)?;
 
-    position.lp_fee_payments = position.lp_fee_payments
+    position.lp_fee_payments = position
+        .lp_fee_payments
         .checked_add(metrics.fee_payment)
         .ok_or_else(math_error!())?;
 
     // market gets it when its too small
     assert!(metrics.unsettled_pnl <= 0);
-    market.amm.total_fee_minus_distributions = market.amm.total_fee_minus_distributions
+    market.amm.total_fee_minus_distributions = market
```

Comment:
should this go into the markets position pnl? / should we include this in total_exchange_fees for paying funding/repeg?